zsh-history-substring-search.zsh

```diff
@@ -183,7 +184,7 @@ if [[ $+functions[_zsh_highlight] -eq 0 ]]; then
                             zle -N $cur_widget _zsh_highlight_widget_$cur_widget";;
 
         # Builtin widget: override and make it call the builtin ".widget".
-        builtin) eval "_zsh_highlight_widget_$cur_widget() { builtin zle .$cur_widget -- \"\$@\" && _zsh_highlight }; \
+        builtin) eval " _zsh_highlight_widget_$cur_widget() { builtin zle .$cur_widget -- \"\$@\" && _zsh_highlight }; \
```

Comment:
That's not a sufficient reason to merge this particular change; :sweat: please omit it from your PR.